app/routes/application.js

```diff
@@ -130,6 +130,18 @@ export default class ApplicationRoute extends Route.extend(ApplicationRouteMixin
       // Do not save the url of the transition to login route.
       if (!url.includes('login') && !url.includes('reset-password')) {
         this.set('session.previousRouteName', url);
+
+        if (url.includes('/e/')) 
+        {
+          this.set('session.Passage', '1')
+        } 
+        else 
+        {
+          this.set('session.Passage', '1')
+        }
+        
+      } else {
+        this.set('session.Passage', '/login');
```

Comment:
What does this even do?

Reply:
I used it to see that whether someone is trying to login through Login page or he/she is redirected to login page.

Reply:
Will this method work ?

Reply:
You have written this. How in the world would I know?